distributed/scheduler.py

```diff
@@ -4048,7 +4329,7 @@ def identity(self, comm=None):
             "started": self.time_started,
             "workers": {
                 worker.address: worker.identity()
-                for worker in parent._workers_dv.values()
+                for worker in self.state.workers.values()
```

Comment:
These had to change as we get an `AttributeError` when using `._*` attributes that are only exposed in C of a non-base `class`.

Reply:
I meant the difference between "workers" and "workers_dv" with the extra suffix. Are they the same?

Reply:
They are the same object. `_workers` is a `SortedDict`. As `SortedDict` inherits from `dict` and some non-mutating operations (`__getitem__`, `len`, etc.) are faster with `dict` (particularly when Cythonized as they use the C Python API), we cast `_workers` to a `dict` and store that in `_workers_dv`, which is then used for non-mutating operations.

Reply:
OK, I guess. 
These cases seem to be changing `._workers_dv` to `state.workers` which gets `state._workers` - but there is no property exposing `state._workers_dv`, which is the `dict` version.

Reply:
If this code will be pure Python instead of Cython, it won't know any difference. That said, there is a question of whether it is worth refactoring this logic out into `SchedulerState` to benefit from the faster access in Cython/C. It's unclear to me whether we need to do that here yet